common_docs_zh/ecosystem/kettle.md

```diff
@@ -0,0 +1,81 @@
+---
+{
+    "title": "Kettle Plugin",
+    "language": "zh-CN"
+}
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Kettle doris stream load插件
+
+[Kettle](https://pentaho.com/) doris stream load 插件，用于在Kettle中通过 Stream Load 将其他数据源的数据写入到 Doris 中。
```

Comment:
please capitalize Doris , Stream Load , and add blank between Chinese and English